### PR TITLE
Sharing the simple pleasure of an often forgotten old command

### PR DESCRIPTION
**Pull request by unlucio**

### CODE REVIEW

**Comment by unlucio** [comment]
@p16 @davidino

**Comment by davidino** [comment]
:+1:  cool

**Comment by p16** [comment]
I'm moved! :')

File: source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+With the "power of the Internet" you've all a bunch of possible way to do it with a simple journey around the planet:
+- send an email
+- put it on dropbox/gdrive/box/etc
+- skype/facebookMessanger/*chat*/*IM*
+- ...
+
+After all, who doesn't have an internet connection these days?
+
+I'm an old guy and back when I was in college having an internet connection wasn't that obvious, but you could almost always count on a 
+[cat 5](http://en.wikipedia.org/wiki/Category_5_cable) [twisted pair](http://en.wikipedia.org/wiki/Twisted_pair) with an [rj45](http://en.wikipedia.org/wiki/RJ45) plug. Why? Simple: stumbling on a sudden [LAN party](http://en.wikipedia.org/wiki/LAN_party), or just crave a quick [frag](http://slanggeek.com/Slang/Details/frag) with a pal in your fav map on [Quake3](http://en.wikipedia.org/wiki/Quake_III_Arena) or [Unreal](http://en.wikipedia.org/wiki/Unreal_Tournament) was way more common than finding a USB stick of a wifi connection in an age they didn't yet exists.
```
Comment (p16) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: "of a wifi" => "or a wifi"

Comment (p16) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: exists => exist

File: source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+title: "Seding files in unix: the quick and dirty way"
+date: 2015-04-19 09:39
+comments: true
+categories: [unix, shell, network, command, vintage]
+author: Luciano Colosio
+---
+
+You're hacking around with a friend, and there you need to send him a file. How would you do that?
+
+Just use a UBS stick (do people still carry those things around?).
```
Comment (razanbilwani) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: fedex

File: source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Second option: we have [SSH](http://en.wikipedia.org/wiki/Secure_Shell)! It's secure, and comes with [scp](http://en.wikipedia.org/wiki/SCP)!<br>
+Great! <br>
+NO! <br>
+No way I'd give remote shell access to someone to my laptop. Right?
+
+FTP, WEBDAV, and a plethora of other options could have probably done it, sure. With configuration or giving account on your machine... See what I mean?
+
+Today probably some young entrepreneur would come up with a whole startup to solve this problem, our simple solution use to be: [netcat](http://en.wikipedia.org/wiki/Netcat), the TCP/IP swiss army knife.
+
+NC (short for netcat) can open tcp sockets for both reading (listen) an writing:
```
Comment (p16) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: an => and

File: source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown (hunk 2 above)
Comment (p16) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: lol

Comment (unlucio) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: LOL

File: source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+nc -l 1234 | tar x
+```
+
+sender:
+```bash
+tar -c <your dir name> | nc <host> 1234
+```
+And that's all :)
+
+PS: thanks to [David](/team#David Funaro) and [Philo](/team#Filippo De Santis) for reminding me about those days when learning and using old tools was simpler than making a new company ;P
```
Comment (odino) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: Philo or Filo? :)

Comment (unlucio) [inline source/_posts/2015-04-19-seding-files-in-unix-the-quick-and-dirty-way.markdown]: I'm sneakily changing his nick ;)

**Comment by unlucio** [comment]
@p16 can I call you phil?

Can i publish this?

**Comment by p16** [comment]
@unlucio yes :)

**Comment by odino** [comment]
@unlucio go ahead, you can / should publish this :) though remember to rebase as ive published a blog post some time back and im not sure it is included in this branch

**Comment by unlucio** [comment]
online http://tech.namshi.com/blog/2015/04/19/seding-files-in-unix-the-quick-and-dirty-way/

**Comment by odino** [comment]
@unlucio can merge this

**Pull request merged**